cfbs/masterfiles/analyze.py

```diff
@@ -120,3 +120,9 @@ def version_as_comparable_list_negated(version):
     vcl[1] = [-x for x in vcl[1]]
 
     return vcl
+
+
+def version_is_at_least(version, min_version):
```

Comment:
It would be nice to have a couple of versions compare functions in `utils.py` for the different comparison operators `>` , `<` , `>=` , `<=` , `==` , and `!=` along with unit tests. This is probably out of the scope of this PR. But feel free to do that in a follow-up PR, if you are up for the challenge :wink: I believe we compare versions in multiple places throughout this project. `cfbs add` is one example.